Example/Auth/Tests/FIRAuthAppDelegateProxyTests.m

```diff
@@ -376,12 +376,12 @@ - (void)testEmptyDelegateOneHandler {
     XCTFail(@"Should not call completion handler.");
   }];
   if (_isIOS9orLater) {
-    XCTAssertFalse([delegate application:_mockApplication openURL:_url options:@{}]);
+    XCTAssertNoThrow([delegate application:_mockApplication openURL:_url options:@{}]);
```

Comment:
What's the reason for changing this, was it an existing bug?

Reply:
@ryanwilson I tried to explain it in the commit message (see the [commit](https://github.com/firebase/firebase-ios-sdk/pull/2450/commits/b51c7cf7bbb8d696fa809f868e7db8a8dfebc5b7) ). 

I was able to reproduce the issue only in unit tests only when the code coverage is enabled. Taking into account that:
- a proper fix for that requires significant changes to `FIRAuthAppDelegateProxy`
- the existing implementation is working and has been tested
- the condition does not look like something can happen in a real application
I decided to just fix the test to avoid unnecessary  and possibly risky changes.

Please let me know if you need more details or have any concerns.

Reply:
Ah sorry I missed the commit message. SGTM thanks, but would be nice to have Auth team (@renkelvin) to confirm that's not going to cause any issues.

Reply:
+1 for confirmation from Auth team. Thanks for fixing this though!

Reply:
Changing to `XCTAssertNoThrow` LGTM. Thanks for fixing!